tests/test_enterprise/api/test_views.py

```diff
@@ -8477,3 +8478,17 @@ def test_sso_configuration_delete_not_found(self):
+        entityId = fetch_entity_id_from_metadata_xml(metadata_xml)
+        assert entityId == "https://login.rz.rwth-aachen.de/shibboleth"
```

Comment:
nit: the strings to the right should be saved in a variable named "expected_entity_id"